subprojects/docs/src/docs/stylesheets/userGuideHtml.xsl

```diff
@@ -67,235 +67,134 @@
-                                    <li><a href="/userguide/eclipse_plugin.html">Eclipse Plugin</a></li>
-                                    <li><a href="/userguide/idea_plugin.html">IDEA Plugin</a></li>
+                            <li><a class="nav-dropdown" data-toggle="collapse" href="#sample-gradle-builds" aria-expanded="false" aria-controls="sample-gradle-builds">Sample Gradle builds</a>
+                                <ul id="sample-gradle-builds">
+                                    <li><a href="https://github.com/gradle/gradle/tree/master/subprojects/docs/src/samples">Groovy DSL samples</a></li>
+                                    <li><a href="https://github.com/gradle/kotlin-dsl/tree/master/samples">Kotlin DSL samples</a></li>
                                 </ul>
                             </li>
-                            <li><a href="/userguide/embedding.html">Tooling API</a></li>
+                            <li><p></p></li>
+                            <li><a href="/userguide/feature_lifecycle.html">Gradle feature lifecycle</a></li>
                         </ul>
 
+                        <h3 id="third-party-integration"><a href="/userguide/third_party_integration.html">Gradle &amp; third-party tools</a></h3>
                         <h3 id="developing-gradle-plugins">Extending Gradle</h3>
                         <ul>
-                            <li><a class="nav-dropdown" data-toggle="collapse" href="#plugins-tutorials" aria-expanded="false" aria-controls="plugins-tutorials">Plugin Development Guides</a>
-                                <ul id="plugins-tutorials">
-                                    <li><a href="https://guides.gradle.org/designing-gradle-plugins/">Designing Gradle Plugins</a></li>
-                                    <li><a href="https://guides.gradle.org/implementing-gradle-plugins/">Implementing Gradle Plugins</a></li>
-                                    <li><a href="https://guides.gradle.org/testing-gradle-plugins/">Testing Gradle Plugins</a></li>
-                                    <li><a href="https://guides.gradle.org/publishing-plugins-to-gradle-plugin-portal/">Publishing Gradle Plugins</a></li>
-                                </ul>
-                            </li>
-                            <li><a href="https://guides.gradle.org/using-the-worker-api/">Developing Parallel Tasks</a></li>
-                            <li><a href="/userguide/lazy_configuration.html">Lazy Task Configuration</a></li>
-                            <li><a href="/userguide/java_gradle_plugin.html">Plugin Development Plugin</a></li>
-                            <li><a href="/userguide/custom_plugins.html">Writing Custom Plugins</a></li>
+                            <li><a href="/userguide/custom_plugins.html">Writing custom plugins</a></li>
+                            <li><a href="https://gradle.org/guides/?q=Plugin%20Development">Plugin development guides</a></li>
```

Comment:
I wonder if "tutorials" would be usefully consistent

Reply:
Some are topical guides, not tutorials.

Reply:
Aside, I think we should migrate these plugin dev guides to the user manual non-urgently after #5837 is merged, because they'll now have the same infrastructure :)